god-mode.el

```diff
@@ -308,38 +308,34 @@ from the command loop."
   "Interpret god-mode special keys for KEY.
 Consumes more keys if appropriate.
 Appends to key sequence KEY-STRING-SO-FAR."
-  (let ((key-consumed t) (next-modifier "") next-key)
+  (let ((modifier "")
+	(next-key (lambda () (god-mode-sanitized-key-string (read-event key-string-so-far)))))
```

Comment:
Incorrect indentation here:
```suggestion
  (let ((modifier "")
        (next-key (lambda () (god-mode-sanitized-key-string (read-event key-string-so-far)))))
```

Reply:
Oh it looks like this because you've used tabs instead of spaces 😄
You can just use `M-x untabify` on the whole function to fix it up the indentation.